tests/test_client.py

```diff
@@ -37,8 +39,20 @@ def add_mock_detector_creation_response():
     responses.add(responses.POST, api_url('detectors/'), json={'id': 'foobar'}, status=201)
 
 
+def add_mock_detector_train_responses(detector_id):
+    operation_id = OPERATION_ID
```

Comment:
nit: I guess you can inline that field, to follow the same usage as `TEST_POLL_INTERVAL`